client-es5.js

```diff
@@ -14,6 +14,10 @@ const rules = {
 			{
 				"property": "parentElement",
 				"message": "Prefer parentNode to parentElement as Node.parentElement is not supported by IE11."
+			},
+			{
+				"property": "append",
```

Comment:
Per Ed, this would conflict with non-DOM methods that are named "append". It's a pretty generic name. There's of course jQuery, but there's actually a fair number of other classes to consider as well: [Codesearch](<https://codesearch.wmcloud.org/deployed/?q=(%5E%7C%5C.%7C%5Cs)%5B%5E%24%5D%5Cw%2B%5C.append%5C(&i=nope&files=js%24&excludeFiles=jquery%7Cnode_modules&repos=>).

Results include uses in deployed code of Web APIs like URL SearchParams, BlobBuilder, TextRange, FormData, and Fetch Headers. And in our own source code (or vendored) through MobileFrontend View.js, `mw.translate`, D3, Ace InputStream, and more.